edge/pkg/edgehub/edgehub.go

```diff
@@ -22,6 +22,8 @@ const (
 	ModuleNameEdgeHub = "websocket"
 )
 
+var HasTLSTunnelCerts = make(chan bool, 1)
```

Comment:
TunnelTLSCertsReady ?

Reply:
@fisherxu  yes. HasTLSTunnelCerts and DoneTLSTunnelCerts will inform the tunnel that the certificate is ready so that tunnel could load the certificates successfully.  (HasTLSTunnelCerts is for Edgehub and DoneTLSTunnelCerts is for CloudCore)